src/LanguageServer/Protocol/Protocol/Internal/VSInternalDataTips.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+
+namespace Roslyn.LanguageServer.Protocol;
+
+/// <summary>
+/// DataTip tag enum.
+/// </summary>
+[Flags]
+internal enum VSInternalDataTipTags
+{
+    /// <summary>
+    /// Data tip range is a linq expression.
+    /// </summary>
+    LinqExpression = 0x1,
+}
+
+/// <summary>
+/// Class which represents debugger data tip response.
+/// </summary>
+[DataContract]
```

Comment:
remove - data contract attributes don't apply to STJ

Reply:
done.